x/oracle/docs/README.md

```diff
@@ -0,0 +1,259 @@
+# Oracle Setup Documentation
+
+## Install Base Dependencies
+
+```sh
+# Ubuntu
+sudo apt update && sudo apt upgrade -y
+sudo apt-get install make gcc jq git
+```
+
+### Install Go 1.19
+
+```sh
+curl -fsSL https://golang.org/dl/go1.19.3.linux-amd64.tar.gz | sudo tar -xzC /usr/local
+
+# `go version` should output "go version go1.19.3 linux/amd64"
+```
+
+### Prepare Go Paths
+
+```sh
+# Only run this if your go path is not yet setup.
+# Check this with `echo $PATH`
+
+echo "export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin" >> $HOME/.bashrc
+
+# Reload the bashrc file to the current terminal
+source $HOME/.bashrc
+```
+
+### Build Price Feeder
+
+```sh
+cd ~
+git clone https://github.com/CosmosContracts/juno.git
+cd ./juno/price-feeder
+make install
+
+# Copy config to your location for future use & editing
+cp config.example.toml $HOME/.juno/oracle-config.toml
+
+# If you get `cp: cannot create regular file '$HOME/.juno/oracle-config.toml': No such file or directory`, 
+# run: `junod init [moniker]`. Then run the cp command again
+
+price-feeder version # sdk: v0.45.11 and go1.19.*
+# If it is not found, your go path is not set
+```
+
+---
+
+## Screen
```

Comment:
Feel like this should go below `systemd` as I think that's a more robust way of doing it? Do any validators actually use `screen`?

Reply:
I agree systemd is better, but screen is the more beginner friendly way for most (which is why we did not include tmux). This is why I had it on top

Feel most will be between systemd and the docker